src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -60,6 +60,10 @@
 use PrestaShopBundle\Form\Admin\Product\ProductCategories;
 use Product;
 use Tools;
+use Shop;
```

Comment:
We try to not rely on old models in PrestaShop bundle: in every minor release, we refactor this controller to remove piece by piece calls to legacy.